### PR TITLE
0000-01-02-Nikhimat.md.

### PR DESCRIPTION
**Pull request by Nikhitha-Math**
I created a branch, a file and made commit and opened pull request

### CODE REVIEW

File: 0000-01-02-Nikhimat.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```
Comment (Nikhimat) [inline 0000-01-02-Nikhimat.md:4]: First github contributor

File: _posts/0000-01-02-Nikhimat.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (Nikhimat) [inline _posts/0000-01-02-Nikhimat.md:5]: First github contributor

File: _posts/0000-01-02-Nikhimat.md. [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (Nikhitha-Math) [inline _posts/0000-01-02-Nikhimat.md.]: First github contributor

Comment (Nikhitha-Math) [inline _posts/0000-01-02-Nikhimat.md.]: First github contributor

**Pull request merged**